var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -67,7 +67,7 @@ def patch(self):
             )
         for cmake_filename in find(self.stage.source_path, "CMakeLists.txt"):
             filter_file(r'\$\{GLEW_LIBRARIES\}', 'GLEW', cmake_filename)
-        if self.spec.satisfies('@1.0:'):
+        if self.spec.satisfies('@1.0:1.1'):
```

Comment:
If you want 1.1 still to have the patch applied, then, yes, this is correct.

Reply:
Yes, 1.1 needs it but any newer release or the current stuff in the master branch doesn't need it. Thanks a lot!

Reply:
Feel free to merge, all good from my side. (also, I think this should capture releases like `1.1.1`, IIRC, so I hope your next one will be `1.2`)